avivator/src/Avivator.js

```diff
@@ -89,17 +99,18 @@ export default function Avivator(props) {
   const [panLock, togglePanLock] = useReducer(v => !v, true);
   const [isLensOn, toggleIsLensOn] = useReducer(v => !v, false);
   const [channels, dispatch] = useReducer(channelsReducer, initialChannels);
-
+  const [use3d, toggleUse3d] = useReducer(v => !v, false);
```

Comment:
It might be nice to have a custom `useToggle` hook to remove some of this boilerplate:

```javascript
const useToggle = (init = true) => useReducer(v => !v, init);
```

Reply:
This is eliminated in https://github.com/hms-dbmi/viv/pull/420